### PR TITLE
Clone request config and update Node

### PR DESCRIPTION
**Pull request by njmulsqb**
Fix variable assignment, config does not exist.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -111,11 +111,11 @@ ClientApi.prototype.request = async (url, data, format) => {
   try {
     let requestConfig = defaultAxiosConfig;
     if (data) {
-      config.params = { ...config.params, ...data };
+      requestConfig.params = { ...requestConfig.params, ...data };
```
Comment (thc202) [inline src/index.js:114]: Doesn't this cause https://github.com/zaproxy/zap-api-nodejs/pull/29#discussion_r1111799467 again?

Comment (njmulsqb) [reply src/index.js:114]: Oh yes, I forgot the reason behind it but linter showed me that config variable is not defined, that's why I made this change.

Comment (thc202) [reply src/index.js:114]: It could do `let requestConfig = structuredClone(defaultAxiosConfig);`. (Node version needs to be updated to at least 17.)

Comment (njmulsqb) [reply src/index.js:114]: Should we do it?

Comment (thc202) [reply src/index.js:114]: I think so.

Comment (thc202) [reply src/index.js:114]: Node version still needs to be updated.

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Pull request merged**